llvm/utils/lit/tests/Inputs/shtest-cat/cat-e-newline.txt

```diff
@@ -0,0 +1,12 @@
+# RUN: cat -e %S/txt-files/newline | FileCheck %s
```

Comment:
Does this need to be its own file? I understand you're separating out the `-e` behaviors, but maybe we should either group all of the tests based on their flags (e.g. one test file for `cat`, `cat -v`, and `cat -e`), or group them on per-file behavior (e.g. one test for `newlines`, one for `allchars`, etc.). I feel like thats likely to reduce duplication.

Reply:
done